docs/en/integrations/data-visualization/rocketbi-and-clickhouse.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_label: Rocket BI
+sidebar_position: 131
+slug: /en/connect-a-ui/rocketbi-and-clickhouse
+keywords: [clickhouse, rocketbi, connect, integrate, ui]
+description: RocketBI is a self-service business intelligence platform that helps you quickly analyze data, build drag-n-drop visualizations and collaborate with colleagues right on your web browser.
+---
+import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
+
+
+# GOAL: BUILD YOUR 1ST DASHBOARD
+
+In this guide, you will install and build a simple dashboard using Rocket.BI .
+This is the dashboard:
+
+<img width="800" alt="github_rocketbi2" src={require('./images/rocketbi_01.gif').default}/>
+<br/>
+
+You can checkout [the Dashboard via this link.](https://demo.rocket.bi/dashboard/sales-dashboard-7?token=7eecf750-cbde-4c53-8fa8-8b905fec667e)
+
+## INSTALL
+
+Before we go deeper in create an advance dashboard with Rocket.BI, making sure you have rocket.bi platform on your ClickHouse server and the data is connected & up-to-date. Here are the step by step guide on the configuration:
+
+Follow the guide here: [Rocket.BI Readme](https://github.com/datainsider-co/rocket-bi/blob/main/README.md)
```

Comment:
Please provide instructions on setting up a connection between ClickHouse and RocketBI. This content https://github.com/datainsider-co/rocket-bi/blob/main/README.md#get-started is the more challenging part for a newcomer.

Reply:
done, please check.
Basically, I add a simple instruction for people to try it out using docker. 
`wget https://raw.githubusercontent.com/datainsider-co/rocket-bi/main/docker/docker-compose.yml `
`wget https://raw.githubusercontent.com/datainsider-co/rocket-bi/main/docker/.clickhouse.env`
Edit `.clickhouse.env`, add clickhouse server information.
`docker-compose up -d . `